src/test/interface/example_output/mix.nom

```diff
@@ -7,12 +7,10 @@ No divergent transitions found.
 Checking E-BFMI - sampler transitions HMC potential energy.
 E-BFMI satisfactory.
 
-The following parameters had fewer than 0.001 effective draws per transition:
-  mu[1], mu[2], theta
-Such low values indicate that the effective sample size estimators may be biased high and actual performance may be substantially lower than quoted.
+Effective sample size satisfactory.
 
-The following parameters had split R-hat greater than 1.05:
-  mu[1], mu[2], theta
+The following parameters had split R-hat greater than 1.01:
```

Comment:
Should we say `split rank-normalized`...?

Reply:
or rank-normalized split ... ?

` rhat(rank_transform(split_draws_matrix));`

Reply:
following discussion in Vehtari et al, it's "rank normalized split-Rhat"